### ISSUE TITLE
update problem : don't support separate library in new update

### ISSUE THREAD

**Issue by pMonfared** [aspnet/Localization#275]
i'm using localization in separate class library and it worked in 1.0.0
but don't work after updated to 1.1.0-alpha1-21873

```
public class AccountController : Controller
    {
        private readonly IStringLocalizer _stringLocalizer;
        private readonly IHtmlLocalizer _htmlLocalizer;

        private readonly UserManager<ApplicationUser> _userManager;
        private readonly SignInManager<ApplicationUser> _signInManager;
        private readonly IEmailSender _emailSender;
        private readonly ISmsSender _smsSender;
        private readonly ILogger _logger;

        public AccountController(
            IStringLocalizerFactory stringLocalizerFactory,
            IHtmlLocalizerFactory htmlLocalizerFactory,
            UserManager<ApplicationUser> userManager,
            SignInManager<ApplicationUser> signInManager,
            IEmailSender emailSender,
            ISmsSender smsSender,
            ILoggerFactory loggerFactory)
        {
            _stringLocalizer = stringLocalizerFactory.Create(
        baseName: "Controllers.AccountController" ,
        location: "Bookkhan.ExternalResources" );
            _htmlLocalizer = htmlLocalizerFactory.Create(
                baseName: "Controllers.AccountController" ,
                location: "Bookkhan.ExternalResources" );

            _userManager = userManager;
            _signInManager = signInManager;
            _emailSender = emailSender;
            _smsSender = smsSender;
            _logger = loggerFactory.CreateLogger<AccountController>();
        }
...
```

![untitledsss](https://cloud.githubusercontent.com/assets/10448255/17697731/bd31e866-63cb-11e6-8342-a4695d3e61ee.jpg)

**Comment by Eilon** [comment]
Let's see if there's a regression here and if so, we should make sure we add tests before we fix.

**Comment by hishamco** [comment]
@pMonfared do you have the sample in github repo?

**Comment by Eilon** [comment]
@ryanbrandenburg let's see if we can make this work properly once we do the other fix with the attribute.

**Comment by ryanbrandenburg** [comment]
@pMonfared this should be fixed by #283 which is merged into dev now? Can you give `ResourceLocationAttribute` a try and confirm it fixes your situation?

**Comment by ryanbrandenburg** [comment]
On second thought, what you've said here is pretty much the exact bug we fixed, I'll just close this out and you feel free to reopen it if you're still having trouble.
